ArduPlane/Parameters.h

```diff
@@ -401,6 +404,9 @@ class Parameters {
     AP_Int16 waypoint_radius;
     AP_Int16 waypoint_max_radius;
     AP_Int16 rtl_radius;
+    
+    // Thermalling
+    AP_Int16 loiter_radius;
```

Comment:
This needs to be removed. Since v3.2 some params have moved to AP_Vehicle so that other libraries can access it. Here's where it is defined:
https://github.com/ArduPilot/ardupilot/blob/master/ArduPlane/Parameters.cpp#L286
and
https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_Vehicle/AP_Vehicle.h#L46